src/main/java/frc/team3128/Constants.java

```diff
@@ -157,7 +157,7 @@
 
         public static final int HOOD_MOTOR_ID = 5;
 
-        public static final double kP = 0.3128;//0.4;//0.3128;//0.254;
+        public static final double kP = 0.28;//0.4;//0.3128;//0.254;
```

Comment:
The true hood angles we observed were pretty close to 1 degree above setpoint every time - hood PID should be retuned, but not at the cost of the data we collected today